README.md

```diff
@@ -225,6 +225,41 @@ ping_type = "udp"
 
 [HCL]: https://github.com/hashicorp/hcl "HashiCorp Configuration Language (HCL)"
 
+### Consul ACL Policies
+
+With [ACL system][ACL] enabled on Consul agents, a specific ACL policy may be
+required for ESM's token in order for ESM to perform its functions. To narrow
+down the privileges required for ESM the following [ACL policy rules][rules]
+can be used:
+
+```hcl
+agent_prefix "" {
+  policy = "write"
+}
+
+key_prefix "consul-esm/" {
+  policy = "write"
+}
+
+node_prefix "" {
+  policy = "read"
```

Comment:
```suggestion
  policy = "write"
```

Reply:
I discovered a few APIs that use node policies. `/coordinate/update`, amongst others, used [here](https://github.com/hashicorp/consul-esm/blob/ef82992b1097d1a59f27bcc96f152e8de7bdb7a3/coordinate.go#L368
). It requires a [`node:write`](https://www.consul.io/api-docs/coordinate#update-lan-coordinates-for-a-node
)